### ISSUE TITLE
Sidebar: Close button is missing discernible text

### ISSUE THREAD

**Issue by WTW-Steve-Martin** [primefaces/primeng#14838]
### Describe the bug

The Axe Dev Tools for Accessibility is reporting that the close button for the `p-sidebar` doesn't have any discernible text.
![image](https://github.com/primefaces/primeng/assets/153538315/342e8291-2099-4715-ad34-40ebc13d63e3)

### Environment

Windows 10

### Angular version

17.2.1

### PrimeNG version

17.7.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Browser(s)

Chrome and Edge

### Steps to reproduce the behavior

1. Install the `Axe Dev Tools for Accessibility` plug-in.
2. Open primeng sidebar page (https://primeng.org/sidebar) on Chrome/Edge browser.
3. Scan the page with Axe

### Expected behavior

The sidebar passes all Axe tests.

**Comment by maruthumj** [comment]
@WTW-Steve-Martin I will start working on this one

**Comment by WTW-Steve-Martin** [comment]
Thanks very much

**Comment by WTW-Steve-Martin** [comment]
I'm afraid adding the `ariaLabel` property to the `p-sidebar` did not change it's associated close button.
![image](https://github.com/primefaces/primeng/assets/153538315/bf2c49a4-46bf-4a8f-b9cc-b1230afb8079)

**Comment by maruthumj** [comment]
@WTW-Steve-Martin sorry didn't notice it was a sidenav. you can pass the aria-label by using "ariaCloseLabel" property from sidenav. I hope this will resolve your issue!

**Comment by WTW-Steve-Martin** [comment]
That solved it. Thanks very much
